README_B2C.md

```diff
@@ -0,0 +1,167 @@
+---
+page_type: sample
+languages:
+- python
+- html
+products:
+- azure-active-directory
+description: "This sample demonstrates a Python web application calling a Microsoft Graph that is secured using Azure Active Directory."
+urlFragment: ms-identity-python-webapp
+---
+# Integrating B2C feature of Microsoft Identity Platform with a Python web application
```

Comment:
>Identity Platform [](start = 39, length = 17)

I learned at //build that this is not capitalized -> Microsoft identity platform . lol. #marketing